sdk/src/test/java/io/customer/sdk/CustomerIOTest.kt

```diff
@@ -0,0 +1,182 @@
+package io.customer.sdk
+
+import io.customer.base.data.ErrorResult
+import io.customer.base.error.ErrorDetail
+import io.customer.base.error.StatusCode
+import io.customer.base.utils.ActionUtils.Companion.getEmptyAction
+import io.customer.base.utils.ActionUtils.Companion.getErrorAction
+import io.customer.sdk.data.model.Region
+import io.customer.sdk.data.request.MetricEvent
+import io.customer.sdk.utils.verifyError
+import io.customer.sdk.utils.verifySuccess
+import org.amshove.kluent.`should be equal to`
+import org.junit.Before
+import org.junit.Test
+import org.mockito.Mockito
+import org.mockito.kotlin.any
+
+internal class CustomerIOTest {
+
+    private lateinit var customerIO: CustomerIO
+    lateinit var mockCustomerIO: MockCustomerIOBuilder
+
+    @Before
+    fun setUp() {
+        mockCustomerIO = MockCustomerIOBuilder()
+        customerIO = mockCustomerIO.build()
+    }
+
+    @Test
+    fun `verify SDK configuration is correct`() {
+        customerIO.config.siteId `should be equal to` "mock-site"
+        customerIO.config.apiKey `should be equal to` "mock-key"
+        customerIO.config.timeout `should be equal to` 6000
+        customerIO.config.region `should be equal to` Region.US
+        customerIO.config.urlHandler `should be equal to` null
```

Comment:
I prefer to avoid hard-coded values in tests because if, for example, `MockCustomerIOBuilder.siteId` changes to `"mock-site-id"` by someone in the future, this test breaks. 

What about something like this? 

```kt
    fun `verify SDK configuration is set to default values`() {
        customerIO.config.siteId `should be equal to` MockCustomerIOBuild.siteId
        ...
```